slave/slave.go

```diff
@@ -4,25 +4,28 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/http"
+	"strconv"
 
 	"github.com/aaronang/cong-the-ripper/lib"
 )
 
 var slaveInstance Slave
 
 type Slave struct {
+	port        int
 	heartbeat   lib.Heartbeat
 	successChan chan CrackerSuccess
 	failChan    chan CrackerFail
 	addTaskChan chan lib.Task
 }
 
-func Init(instanceId string) *Slave {
+func Init(instanceId string, port int) *Slave {
 	heartbeat := lib.Heartbeat{
 		SlaveId: instanceId,
 	}
 
 	slaveInstance = Slave{
```

Comment:
No it cannot, since the `func taskHandler(w http.ResponseWriter, r *http.Request)` needs to access slaveInstance.